questions.md

```diff
@@ -0,0 +1,446 @@
+
+Some of these questions will fell _very_ "_personal_".
+The answers are yours to keep and reflect on, be honest with _yourself_.
+
+The questions are _meant_ to make you _think_.
+Write down the _first_ thing that comes into your mind,
+your _first_ thought is usually the most accurate reflection
+of what you are truly feeling.
+
+It should take you one "pomodoro" (_25 minutes_) to write your answers.
+Don't spend more than a few minutes on each question
+otherwise you are "_over-thinking_" it.
+The point is not to write an "essay" for each answer,
+just a few sentences is more than "enough".
+
+There are no "right" or "wrong" answers,
+these are _your_ answers,
+they are "_right_" for _you_
+at this point in your life.
+
+
+These are _personal_ questions we will be asking all our future
+"_core_" team members. It's ~~important~~ _essential_ to have answers.
+
+@dwyl We don't believe in having _formal interviews_
+with "_brainteaser_" questions like <br />
+"How Many [Things] Are There in [Location]?" <br />
+e.g: "How many yoga teachers are there in the USA?" <br />
+or "How many taxi drivers are there in New York City?" <br />
+These types of questions have their place and are useful
+deductive reasoning exercises.
+They are insightful for understanding _how_ people think
+and systematically approach challenges they haven't encountered before.
+But they have limited relevance to understanding _why_ people think
+a certain way or _what_ people think about most often.
+We are _way_ more interested in people's
+answers to the "_Why?_" questions than _how_
+they implement a particular algorithm on a whiteboard.
+We feel that knowing _who_ you are as a _person_
+is the most important thing in a working and friendship relationship.
+
+With that in mind here are the 20 "intro" questions:
+(_the first question is "logistical"
+and helps us improve our efforts to find like-minded people. Thanks!_)
+
+
+## 0. _How_ did you discover dwyl?
+
+```sh
+# I discovered dwyl:
+
+
+
+
+
+
+```
+
+> **Meta**: we love knowing how, where, when and why people discover dwyl.
+It helps us know works in spreading our mission.
+
+
+## 1. _Why_ are you here?
+
+```sh
+# the *reason* I am here today is:
+
+
+
+
+
+```
+
+> **Meta**: this is for _you_ to understand your _own_
+motives for being here.
+
+
+## 2. Is there _anywhere_ you would _rather_ be or _anything_ you would prefer to be doing _right now_?
+
+Is there a ***place*** you would rather be,
+***person*** you would prefer to be with,
+or ***activity*** you would rather be doing?
+
+
+```sh
+# I would rather be ...
+
+
+
+
+```
+
+### 2.a If you would rather be somewhere `else`, _why_ is that?
+
+
+```sh
+# I would rather be ... *because*:
+
+
+
+
+```
+
+> **Meta**: the purpose of this question is for you to _understand_
+what your **priority** is *long-term* so you can ensure to focus on it.
+i.e. "DWYL: Don't Waste Your Life!"
+
+
+## 3. Is there a "significant person" (_or people_) in your life?
```

Comment:
There may be some questions around why _we_ need to see the answers to this as opposed to just a list for 'personal use'.

Reply:
@iteles we will be creating a dynamic decision tree questionnaire in the future where questions can be answered in as much/detail as the person desires or _skipped_.
We will demonstrate through our Open Source Code that the answers are encrypted and _personal_ to the user. If the person choses to _share_ a specific answer with other people e.g: family members, friends or colleagues, they can.